### DISCUSSION TITLE
[Feature Request]: option to make column with empty heading

### DISCUSSION THREAD

**Discussion by amshehzad** [rappasoft/laravel-livewire-tables#1482] [Feature Requests]
### Overview

Sometimes we don't want to have the td value, for example for the actions. 
right now I do it like this: 
`Column::make('')->label(fn($row) => view('includes.datatable.invoice.menu')->withRow($row))`

I think we can make the title nullable, it'll be more readable

`Column::make()->label(fn($row) => view('includes.datatable.invoice.menu')->withRow($row))`

**Comment by lrljoe** [comment]
To avoid ColumnSelect etc conflicts, thought is to add

public bool $hideLabel = false; to cols trait

Then
hideLabel() configuration option to a Column.

Then
isLabelHidden as a helper method for column

Finally
Check if label hidden prior to displaying

Not something that's been asked for previously, and could be achieved with existing methods, but worth implementing.

**Comment by lrljoe** [comment]
It's in progress, but I need to write a few tests for it, and update the docs.  Expect this maybe  next week!

**Comment by lrljoe** [comment]
https://discord.com/channels/1118369537155399690/1162844763129655366
